chromium_src/components/remote_cocoa/app_shim/native_widget_ns_window_bridge.mm

```diff
@@ -52,4 +77,42 @@
   }
 }
 
+void NativeWidgetNSWindowBridge::UpdateWindowTitleColor(SkColor color) {
+  static std::unique_ptr<base::apple::ScopedObjCClassSwizzler>
+      frameClassSwizzler;
+
+  NSView* frameView = window_.contentView.superview;
+  if (![frameView isKindOfClass:[BrowserWindowFrame class]]) {
+    return;
+  }
+
+  if (![frameView respondsToSelector:@selector(_currentTitleColor)]) {
+    return;
+  }
+
+  g_brave_title_color = skia::SkColorToDeviceNSColor(color);
+
+  // Revert previous swizzling and cache original title color.
+  // That title color will be used for non browser window frame.
+  // (ex, task manager dialog or popup window)
+  frameClassSwizzler.reset();
+  NSInvocation* invocation = [NSInvocation
```

Comment:
Don't use `NSInvocation` unless calling functions defined as blocks. Use messaging.
There is a much simpler and safer way to override Objective-C methods and to call methods without `NSInvocation`.

Replace this entire function with:
```ObjC
void NativeWidgetNSWindowBridge::UpdateWindowTitleColor(SkColor color) {
  NSView* frameView = window_.contentView.superview;
  if (![frameView isKindOfClass:[BrowserWindowFrame class]]) {
    return;
  }

  if (![frameView respondsToSelector:@selector(_currentTitleColor)]) {
    return;
  }

  g_brave_title_color = skia::SkColorToDeviceNSColor(color);

  // Cache original title color
  g_original_title_color = [(BrowserWindowFrame*)frameView originalTitleColor];

  // Reset title to apply new title color.
  NSString* title = window_.title;
  window_.title = @"";
  window_.title = title;
}
```